collectors/perf.plugin/perf_plugin.c

```diff
@@ -527,6 +529,38 @@ static void perf_send_metrics() {
 
     // ------------------------------------------------------------------------
 
+    if(likely(perf_events[EV_ID_INSTRUCTIONS].updated) && likely(perf_events[EV_ID_CPU_CYCLES].updated)) {
+        if(unlikely(!ipc_chart_generated)) {
+            ipc_chart_generated = 1;
+
+            printf("CHART %s.%s '' '%s' 'instructions/cycle' %s '' line %d %d %s\n"
+                   , RRD_TYPE_PERF
+                   , "instructions_per_cycle"
+                   , "Instructions per Cycle(IPC). An IPC < 1.0 likely means memory bound, and an IPC > 1.0 likely means instruction bound. "
+                   , RRD_FAMILY_HW
+                   , NETDATA_CHART_PRIO_PERF_IPC
+                   , update_every
+                   , PLUGIN_PERF_NAME
+            );
```

Comment:
It is `[update_every [options [plugin [module]]]`. It looks like you are missing options (`''`) before "plugin".

Reply:
You are right, we are missing this not only for this chart, but for the whole plugin. 
I think this is subject for another PR, because I copied and changed from other chart that the plugin plots.

Reply:
I don't get it. Why do not fix it now if you agree?

Reply:
> An IPC < 1.0 likely means memory bound, and an IPC > 1.0

What about 1.0 btw? 😄

Reply:
>  I don't get it. Why do not fix it now if you agree?

I will rename the PR name, update the description and update it here  :smile: .


> What about 1.0 btw? smile

Good point, this was not defined in the [source](https://www.brendangregg.com/blog/2017-05-09/cpu-utilization-is-wrong.html). I will add it here for users get more details.

Reply:
> I will rename the PR name, update the description and update it here

Ok, let's do not do it. Perhaps I couldn't word my requests properly, I don't think it requires renaming. Let's do that change in another PR.

Reply:
@ilyam8 as I said to you, I brought more commits and I updated the `title`, I added the missing `options` and finally I added the missed `module`. I will update the issue with that has missing `modules` for charts.